### DISCUSSION TITLE
[Feature Request] Higher PL1/2 Values for Overclocking

### DISCUSSION THREAD

**Discussion by Sloreo1131** [seerge/g-helper#2229] [Ideas]
Im currently trying to see how far I can push my [ water cooled Scar 16](https://linustechtips.com/topic/1546110-water-cooling-a-scar-16-409013980hx-laptop/) in R23 but run into PL2 for the entire run at 175W. Im wondering if it is possible to bump PL1/2 higher than the 175w limit in G Helper & Armory Crate to something arbitrarily high, like 250w in Throttle Stop, to let it just send it until thermal throttling? Or is it capped in the BIOS so there would be no point?

Currently I can run the R23 10 minute test and hit 35k at 84* without the laptop fans on at all and really hope to see how far it can go with more power.

**Comment by seerge** [comment]
@Sloreo1131 i have increased max slider value here to 200
[GHelper.zip](https://github.com/seerge/g-helper/files/14620003/GHelper.zip)

P.S. I don't suggest to set values higher than defined by manufacturer and I'm not responsible for any damage caused by your experiments.

**Reply by Sloreo1131** [reply]
Thank you very much for the help. Fully understand, if anything‘s gonna cause issues the laptop it’ll be the GPU shunt mod, or the water cooling, definitely can’t even begin to try to blame anyone other than myself! Thank you again.

**Reply by Sloreo1131** [reply]
Just did a few R23 runs and can confirm both that the PL1/2 increase took effect, and that it is stopped by other limitations. Each run still peaked at 175w but got VR Current and EDP other limits on each run, but never PL2. Thank you again for the help, even if it just ran into another wall.

**Reply by IceStormNG** [reply]
Well, this was expected. ASUS has an EC to control power and they have hard limits baked into hardware which cannot be circumvented.

Mainly to protect the power stages and also the power supply itself.
